src/main/surfrdUtilsMod.F90

```diff
@@ -111,7 +113,7 @@ subroutine convert_cft_to_pft( begg, endg, cftsize, wt_cft )
     !        Convert generic crop types that were read in as seperate CFT's on
     !        a crop landunit, and put them on the vegetated landunit.
     ! !USES:
-    use clm_instur      , only : wt_lunit, wt_nat_patch, fert_cft
+    use clm_instur      , only : wt_lunit, wt_nat_patch  ! fert_cft unnecessary
```

Comment:
If you find something not needed just remove it. Don't worry about a comment.

Reply:
Done. (Initially I kept it commented out in case removing it was a mistake...)